lib/copyable/copyable_extension.rb

```diff
@@ -59,6 +59,13 @@ def copyable(&block)
               # for this brand new model, visit all of the associated models,
               # making new copies according to the instructions in the copyable
               # declaration
+
+              if options[:skip_associations].present?
+                # To skip an association, change its advice to :do_not_copy
+                options[:skip_associations].each do |association|
+                  main.association_list[association] = :do_not_copy if main.association_list.keys.include?(association)
+                end
+              end
```

Comment:
So, I notice that pretty much everything else here in the eloquently named `do_the_copy` block calls code that's elsewhere, rather than actually doing stuff. Is it appropriate for the `skip_associations` implementation to live here, or should it, for aesthetics' sake, be passed through to something else?